x/ccv/provider/client/proposal_handler.go

```diff
@@ -193,7 +193,7 @@ type ConsumerAdditionProposalReq struct {
 func ParseConsumerAdditionProposalJSON(proposalFile string) (ConsumerAdditionProposalJSON, error) {
 	proposal := ConsumerAdditionProposalJSON{}
 
-	contents, err := ioutil.ReadFile(filepath.Clean(proposalFile))
```

Comment:
What does this change do?

Reply:
https://pkg.go.dev/io/ioutil is deprecated